### PR TITLE
18 - add institutional representative section to application forms

### PR DESCRIPTION
**Pull request by caravinci**
Now this one is literally just adding new stuff. Pretty much copy paste of the applicant info section, since no new functionality is added.

### CODE REVIEW

File: components/pages/Applications/ApplicationForm/Forms/Representative.tsx [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+import DoubleFieldRow from './DoubleFieldRow';
+import RequiredFieldsMessage from './RequiredFieldsMessage';
+import {
+  FormSectionValidationState_Representative,
+  FormSectionValidatorFunction_Origin,
+} from './types';
+import { useLocalValidation } from './validations';
+import { transformToSelectOptions } from './validations/helpers';
+
+const Representative = ({
```
Comment (caravinci) [inline components/pages/Applications/ApplicationForm/Forms/Representative.tsx:22]: new JSX

File: components/pages/Applications/ApplicationForm/Forms/types.ts [hunk 2]
```diff
@@ -74,7 +74,21 @@ export type FormSectionValidationState_Introduction =
 export type FormSectionValidationState_IT = FormSectionValidationState_SectionsGenericType<{}>;
 export type FormSectionValidationState_Project = FormSectionValidationState_SectionsGenericType<{}>;
 export type FormSectionValidationState_Representative =
-  FormSectionValidationState_SectionsGenericType<{}>;
+  FormSectionValidationState_SectionsGenericType<{
```
Comment (caravinci) [inline components/pages/Applications/ApplicationForm/Forms/types.ts:77]: field types

File: components/pages/Applications/ApplicationForm/Forms/validations/schemas.ts [hunk 3]
```diff
@@ -64,11 +64,40 @@ export const introductionSchema = yup.object().shape({
   agreement_accepted: yup.boolean().default(false).oneOf([true]).required(),
 });
 
+export const representativeSchema = yup.object().shape({
```
Comment (caravinci) [inline components/pages/Applications/ApplicationForm/Forms/validations/schemas.ts:67]: field validation schemas

File: components/pages/Applications/ApplicationForm/Forms/constants.tsx [hunk 4]
```diff
@@ -11,7 +12,7 @@ export const sectionsData = {
   introduction: { component: Introduction, description: 'Introduction' },
   it: { description: 'F. IT Agreements' },
   project: { description: 'D. Project Information' },
-  representative: { description: 'B. Institutional Representative' },
+  representative: { component: Representative, description: 'B. Institutional Representative' },
```
Comment (caravinci) [inline components/pages/Applications/ApplicationForm/Forms/constants.tsx:15]: and adding the component for the section selector

**Review by caravinci — COMMENTED** [review]
Easy as pie 🥧

**Review by anncatton — APPROVED** [review]
👍

**Pull request merged**